lti_consumer/tests/unit/test_lti.py

```diff
@@ -0,0 +1,287 @@
+# -*- coding: utf-8 -*-
+"""
+Unit tests for lti_consumer.lti module
+"""
+
+import unittest
+
+from mock import Mock, PropertyMock, patch
+
+from lti_consumer.tests.unit.test_utils import make_request
+from lti_consumer.tests.unit.test_lti_consumer import TestLtiConsumerXBlock
+
+from lti_consumer.lti import parse_result_json, Lti1Consumer, Lti2Consumer
+from lti_consumer.exceptions import LtiError
+
+
+INVALID_JSON_INPUTS = [
```

Comment:
I think you should refactor this into dicts, with your comments as strings. That way, if more are introduced, they will be listed that way. Just makes it clearer why they should be failing. It can stand in for test case name, basically, since these are being organized into an array rather than test methods.